src/bin/tools/jar.ts

```diff
@@ -69,19 +69,21 @@ export default async function jar({ groupId, artifactId, version, rootPath, targ
             }
         });
 
-    /**
-     * Create an async pipeline, wait until everything is fully processed
-     */
-    await pipeline(
+    await mkdir(dirname(targetPath), { recursive: true });
+
+    // Create an async pipeline, wait until everything is fully processed
```

Comment:
Okay  this block is super strange. While double-checking the behavior, I noticed that the final success banner was not printed. I experimented a bit and in fact _nothign_ I tried to print after `await pipeline(...)` ever got printed, though the produced jar file was fine, and there was no error thrown anywhere. Will need to investigate this, but reverting it to a bit more naive implementation using a hand-crafter promise yielded the desired result.